### PR TITLE
Image promotion for distroless-iptables v0.2.1

### PR DESCRIPTION
**Pull request by cpanato**
Image promotion for build-image v0.2.1
This is an automated PR generated from `kpromo`
```
kpromo pr --fork cpanato --interactive --project build-image --tag v0.2.1
```

promote `distroless-iptables` that was bump here https://github.com/kubernetes/release/pull/2920

/hold
cc: @kubernetes/release-engineering

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-build-image/images.yaml [hunk 1]
```diff
@@ -400,6 +400,22 @@
...
+  dmap:
+    "sha256:8c8800b9cd50c42a283646edf29bc973deae7854043957769141768c9b2fbabb": ["v0.2.1"]
+- name: distroless-iptables-arm64
+  dmap:
+    "sha256:cdd375dc1323bb97880593917dad9c100b06ecf35ce60e75970bac6d38915e12": ["v0.2.1"]
+- name: distroless-iptables-ppc64le
+  dmap:
+    "sha256:913b9f293917718e1435827802b590d42de987146fb3cc6726351a39700b1d1e": ["v0.2.1"]
+- name: distroless-iptables-s390x
+  dmap:
```
Comment (xmudrii) [inline k8s.gcr.io/images/k8s-staging-build-image/images.yaml:417]: Do we need those actually?

Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-build-image/images.yaml]: already dropped

**Review by saschagrunert — APPROVED** [review]

**Review by xmudrii — APPROVED** [review]

**Pull request merged**